plugin/Field/Collection/FieldLoaderGeneric.php

```diff
@@ -80,9 +80,9 @@ public function load(): Generator
 			if (isset($fieldArray['label'])) {
 				unset($fieldArray['label']);
 			}
-
+			$listTypeUnSupported = ['user', 'datei', 'redhint', 'blackhint', 'dividingline'];
 			foreach ($fieldArray as $fieldName => $fieldProperties) {
-				if ($module === onOfficeSDK::MODULE_ADDRESS && $fieldProperties['tablename'] === 'addressFaktura') {
```

Comment:
Is there a reason why this condition got removed? I think this is still necessary

Reply:
Thanks